container.go

```diff
@@ -528,6 +531,18 @@ func (c *Container) stop() error {
 		return err
 	}
 
+	defer func() {
```

Comment:
I don't think this part was needed since the shim is started by `createContainer()` and stopped by `delete()`. It was on purpose that we were stopping the shim in the `delete()`, in order to mirror the create behavior.
I don't think this code part hurt but it kind of duplicates the stop of the shim, and it actually slow down the stop action.